infra/gcp/ensure-main-project.sh

```diff
@@ -48,6 +48,9 @@ CLUSTER_ADMINS_GROUP="k8s-infra-cluster-admins@kubernetes.io"
 # The accounting group.
 ACCOUNTING_GROUP="k8s-infra-gcp-accounting@kubernetes.io"
 
+# The GCS bucket which hold terraform state for clusters
+CLUSTER_TERRAFORM_BUCKET="gs://${k8s-infra-clusters-terraform}"
```

Comment:
This doesn't look right.. why is wrapped like a variable, but has dashes in it?

Reply:
good catch